packages/api/src/controllers/stream.js

```diff
@@ -101,11 +101,19 @@ app.get("/", authMiddleware({ admin: true }), async (req, res) => {
     nonLivepeerOnly,
     order,
     filters,
+    userId,
   } = req.query;
   if (isNaN(parseInt(limit))) {
     limit = undefined;
   }
 
+  if (userId && req.user.admin !== true && req.user.id !== userId) {
```

Comment:
This whole endpoint is accessible only for admins (`authMiddleware({ admin: true })`)
So you should remove `{admin: true}` to make it accessible to regular users.

Reply:
Great! Glad we can easily open up this endpoint to regular users. I'm sure you are already doing this, but I want to make sure that a regular users only has access to their own streams, not everyones. Only admins should see all users' streams.

Reply:
Yup, that is handled.